requirements.txt

```diff
@@ -0,0 +1,4 @@
+wikipedia==1.4.0
```

Comment:
Please sort requirements alphabetically so that when the list gets long it remains easy to spot duplicate or missing requirements.